packages/kotti-ui/source/kotti-avatar/KtAvatar.vue

```diff
@@ -129,19 +128,22 @@ export default defineComponent<KottiAvatar.PropsInternal>({
 
 	&--is-size-small {
 		width: 1.6rem;
-		height: 1.6rem;
+		max-height: 1.6rem;
+		aspect-ratio: 1/1;
```

Comment:
Why repeat this line (`aspect-ratio: 1/1;`) in every size modifier and not move it under `.kt-avatar {`?